packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -382,18 +382,26 @@ class Frame extends React.Component {
     return ariaLabel === undefined ? defaultAriaLabel : ariaLabel;
   }
 
+  /**
+   * Generates the main label for the field.
+   */
+  mainLabel() {
+    let label;
+    if (this.props.display) {
+      label = `${this.props.display} is selected, ${this.ariaLabel()}`;
```

Comment:
string "is selected" needs s translation

Reply:
Used "Terra.form.select.selected" for "selected"  in https://github.com/cerner/terra-core/pull/3705/commits/e9b5080cdba87f0c83cb1bed05064ca38d78882a